airport/airport-pojo-bl/src/main/java/berlin/assets/Asset.java

```diff
@@ -43,25 +42,23 @@
     @Title(value = "Number", desc = "A unique asset number, auto-generated.")
     @CompositeKeyMember(1)
     @Readonly
-    private String number;
-
+    private String number;    
+    
     @IsProperty
     @MapTo
-    @Title(value = "Condition Rating", desc = "Condition rating for an asset")
-    private ConditionRating conditionRating;
+    @Title(value = "Asset Type", desc = "Asset Type of an Asset")
+    private Asset assetType;
```

Comment:
Are you sure that all is good with the type here?

Reply:
fixing it